Release_Protocol.md

```diff
@@ -177,7 +190,7 @@ Verify ReadTheDocs builds complete and publish. If needed, manually
 trigger [builds](https://readthedocs.org/projects/bids-specification/builds/)
 for `stable` and the most recent tag.
 
-### 8. Edit the mkdocs.yml file site_name to set a new development version
+### 9. Edit the mkdocs.yml file site_name to set a new development version
 
 Please submit a PR with the title `REL: <version>-dev`.
 This should be the first merged PR in the new version.
```

Comment:
```suggestion
Please commit to `master` with the title `REL: <version>-dev`.
This should be the first commit in the new version.
```

ref: https://github.com/bids-standard/bids-specification/pull/448#issuecomment-613681797

This is a small enough change we can trust this process